.github/workflows/sync-s3-media.yml

```diff
@@ -0,0 +1,64 @@
+---
+name: Perform Media and Database Backups
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        required: true
+        type: string
+  workflow_call:
+    inputs:
+      environment:
+        required: true
+        type: string
+
+jobs:
+  backup-media:
+    name: Perform Media Backups
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment }}
+    env:
+      space: ${{ inputs.environment }}
+    steps:
+      - name: Backup media files
+        uses: cloud-gov/cg-cli-tools@main
+        with:
+          cf_username: ${{ secrets.CF_USERNAME }}
+          cf_password: ${{ secrets.CF_PASSWORD }}
+          cf_org: gsa-tts-oros-fac
+          cf_space: ${{ env.space }}
+          command: cf run-task gsa-fac -k 2G -m 2G --name s3_sync --command "./s3-sync.sh"
```

Comment:
Because the [backups bucket is shared with multiple environments](https://github.com/GSA-TTS/FAC/blob/ff0a6dce48086caa9d4ddc5e6f5f2c9d25540169/.github/workflows/terraform-apply-env.yml#L75), should `s3-sync.sh` accept an `$ENV` variable so backups can go into environment-specific paths?